eslint.config.js

```diff
@@ -0,0 +1,125 @@
+import jsdoc from "eslint-plugin-jsdoc";
+import globals from "globals";
+import js from "@eslint/js";
+import google from "eslint-config-google";
+
+export default [
+	js.configs.recommended,
+	google,
+	{
+		ignores: ["**/dist/", "**/coverage/", "**/docs/"],
+	},
+	{
+		plugins: {
+			jsdoc,
+		},
+
+		languageOptions: {
+			globals: {
+				...globals.node,
+			},
+
+			ecmaVersion: 2022,
+			sourceType: "module",
+		},
+
+		settings: {
+			jsdoc: {
+				tagNamePreference: {
+					return: "returns",
+				},
+			},
+		},
+
+		rules: {
+			"indent": ["error", "tab"],
+			"linebreak-style": ["error", "unix"],
+
+			"quotes": [
+				"error",
+				"double",
+				{
+					allowTemplateLiterals: true,
+				},
+			],
+
+			"semi": ["error", "always"],
+			"no-negated-condition": "off",
+			"require-jsdoc": "off",
+			"no-mixed-requires": "off",
+
+			"max-len": [
+				"error",
+				{
+					code: 120,
+					ignoreUrls: true,
+					ignoreRegExpLiterals: true,
+				},
+			],
+
+			"no-implicit-coercion": [
+				2,
+				{
+					allow: ["!!"],
+				},
+			],
+
+			"comma-dangle": "off",
+			"no-tabs": "off",
+			// This rule must be disabled as of ESLint 9. It's removed and causes issues when present.
```

Comment:
I thought disabling does not work here? So, manual delete is not necessary?

Reply:
Nope, this one seems to be sufficient

Reply:
Nope, this one seems to be sufficient

Reply:
interesting. good catch.